extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -477,6 +479,9 @@ export class AmpStoryPage extends AMP.BaseElement {
       case PageState.PLAYING:
         if (this.state_ === PageState.NOT_ACTIVE) {
           this.element.setAttribute('active', '');
+          if (this.openAttachmentEl_) {
+            this.openAttachmentEl_.setAttribute('active', '');
+          }
```

Comment:
You need to do this but `removeAttribute` with the other PageState above (where it runs `this.element.removeAttribute`).